lib/src/features/auth/data/repositories/auth_repository_impl.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:community_with_legends_mobile/src/core/errors/exceptions.dart';
+import 'package:community_with_legends_mobile/src/features/auth/data/data_sources/auth_api.dart';
+import 'package:community_with_legends_mobile/src/features/auth/domain/repositories/auth_repository.dart';
+
+class AuthRepositoryImpl implements AuthRepository {
+  final AuthApi api;
+
+  AuthRepositoryImpl(this.api);
+
+  @override
+  Future<String> login(String email, String password) async {
+    final response = await api.login(email, password);
```

Comment:
Please be consistent.
```suggestion
    final response = await api.login(email, password);
    
```